tests/pipelines/test_pipelines_automatic_speech_recognition.py

```diff
@@ -298,6 +298,23 @@ def test_torch_large(self):
         output = speech_recognizer(filename)
         self.assertEqual(output, {"text": "A MAN SAID TO THE UNIVERSE SIR I EXIST"})
 
+    @require_torch
+    @slow
+    def test_torch_large_fbank(self):
```

Comment:
I would love to have a short description of what this test is doing (than the above `test_torch_large`).

By reading this block:
- I don't know what `fbank` is and/or if this test is testing `fbank` or other stuff
  - from the PR description, `fbank` is not mentioned, that's why I get a bit lost

Reply:
I'll cat it `test_torch_large_with_input_features`